.github/workflows/contributor_response.yml

```
@@ -0,0 +1,41 @@
+name: Auto-response to Outside Contributors
+
+on:
+  issue_comment:
+    types: [created] # Ensure it only triggers on comment creation
+
+jobs:
+  auto-response:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Check if commenter is a contributor
+        id: check-if-commenter-is-a-contributor
+        uses: actions/github-script@v6
```

```suggestion
        uses: actions/github-script@v7
```
(blocking) Updating to the latest version